includes/contact_component.inc

```diff
@@ -893,6 +893,10 @@ function wf_crm_find_relations($cid, $types = array(), $current = TRUE) {
       if (!$allowed || in_array($dao->relationship_type_id . '_' . $a_b, $allowed)) {
         $c = $dao->{"contact_id_$a_b"};
         $found[$c] = $c;
+        if ($type_ids && !empty($dao->case_id) && !empty($_GET["case1"]) && $_GET["case1"] == $dao->case_id) {
```

Comment:
If we were in object context then I would recommend using `$this->data['case'][1]` instead of reaching into `$_GET` but I don't think cases have actually been loaded yet.
```suggestion
        // Prefer case roles over other relationships
        if ($type_ids && !empty($dao->case_id) && wf_crm_aval($_GET, 'case1') == $dao->case_id) {
```

Reply:
yes, cases are not loaded as of now, hence fetching it the way it is done to set case objects.